app.ts

```diff
@@ -2,9 +2,16 @@ import { NextFunction, Request, Response } from 'express';
 
 import express from 'express';
 import routes from './routes/api';
+import cors from 'cors';
 
 const app = express();
 
+// currently only setting allowed origin for local dev
+// need to enable cors since the FE makes an API call to the server from a different port
+app.use(cors({
+  origin: /^http:\/\/localhost:[0-9]+/

```

Comment:
doing `cors()` with no options just sets origin to `*` (so every origin is allowed) so not sure if y'all have a preference or just want to restrict it to local dev for now ( I went with the latter but I don't really have a preference)